src/devices/GoPiGo3/Models/SpiMessages.cs

```diff
@@ -11,86 +11,127 @@ public enum SpiMessageType
     {
         /// <summary>None</summary>
         None = 0,
+
         /// <summary>Get manufacturer</summary>
         GetManufacturer,
+
         /// <summary>Get name</summary>
         GetName,
+
         /// <summary>Get hardware version</summary>
         GetHardwareVersion,
+
         /// <summary>Get firmware version</summary>
         GetFirmwareVersion,
+
         /// <summary>Get id</summary>
         GetId,
+
         /// <summary>Set LED</summary>
         SetLed,
+
         /// <summary>Get voltage (5V)</summary>
         GetVoltage5V,
+
         /// <summary>Get voltage (VCC)</summary>
         GetVoltageVcc,
+
         /// <summary>Set servo</summary>
         SetServo,
+
         /// <summary>Set motor power</summary>
         SetMotorPower,
+
         /// <summary>Set motor position</summary>
         SetMotorPosition,
+
         /// <summary>Set motor position controller's proportional factor</summary>
         SetMotorPositionKp,
+
         /// <summary>Set motor position controller's derivative (change) factor</summary>
         SetMotorPositionKd,
+
         /// <summary>Set motor speed (degrees per second)</summary>
         SetMotorDps,
+
         /// <summary>Set motor limits</summary>
-        setMotorLimits,
+        SetMotorLimits,
```

Comment:
One more casing breaking change. I guess since we've taken others like this it would make sense to take this one as well.